### DISCUSSION TITLE
Seeking Technical Insights on NGINX Throughput Variations with Small vs. Large HTTP Requests

### DISCUSSION THREAD

**Discussion by fero-majeeth** [nginx/nginx#300] [General]
I am observing that in the existing NGINX application, issuing small HTTP requests results in more requests per second but less total throughput. Conversely, issuing large HTTP requests results in fewer requests per second but more throughput. I am trying to understand why NGINX results in lower throughput with small HTTP requests. Does anyone know the technical reason behind this?

**Comment by Maryna-f5** [comment]
This discussion appears to focus on the configuration, troubleshooting or general questions around running NGINX, which is not the intended use for GitHub discussions. 

We encourage you to visit [https://community.nginx.org ](https://community.nginx.org/)and share your discussion there as the NGINX Community Forum is the most appropriate place to engage for these types of questions.
